test/intl402/Temporal/PlainDateTime/prototype/toString/calendarname-auto.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.tostring
+description: Possibly display calendar when calendarName is "auto"
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(1976, 11, 18, 15, 23);
+
+assert.sameValue(
+  dt.withCalendar("gregory").toString(),
```

Comment:
Also test with `{ calendarName: "auto" }` passed explicitly, and please add a test for `"always"` with `"gregory"` as well.

Reply:
done